### PR TITLE
[Fix] general

### PR DESCRIPTION
**Pull request by jpfeuffer**
General fixes.
There was a recent change that broke the pipeline with Percolator. We need to introduce tests for this!

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
           sudo mv nextflow /usr/local/bin/
       - name:  BASIC Run the basic pipeline with the test
         run: |
-          nextflow run ${GITHUB_WORKSPACE} "$TOWER" -name "$RUN_NAME-basic" -profile test,docker
+          nextflow run ${GITHUB_WORKSPACE} "$TOWER" -name "wHATEVER" -profile test,docker
```
Comment (Zethson) [inline .github/workflows/main.yml]: :)

Comment (jpfeuffer) [reply .github/workflows/main.yml]: haha sorry, it annoyed me. I asked a question on slack now.

Comment (Zethson) [reply .github/workflows/main.yml]: I saw it :)
I'm just digging the passive aggressive 'wHATEVER'. The lower 'w' is a nice touch haha

Comment (jpfeuffer) [reply .github/workflows/main.yml]: I appreciate it. This will work until we have multiple test configs. We will change it then.

**Pull request merged**